tests/ciscotest.rb

```diff
@@ -100,7 +100,7 @@ def platform
 
   def config_and_warn_on_match(warn_match, *args)
     if node.client.platform == :ios_xr
-      result = super(warn_match, *args, 'commit best-effort')
+      result = super(warn_match, *args, 'commit')
```

Comment:
I tried to remove "best-effort" back in March, and Glenn Matthews said that there was a reason it was needed, so I left it in.  Not sure if it's still needed or not.